src/libs/antares/study/variable-print-info.h

```diff
@@ -104,18 +95,21 @@ class AllVariablesPrintInfo
 public:
     // Public methods
     AllVariablesPrintInfo() = default;
-    ~AllVariablesPrintInfo();
+    ~AllVariablesPrintInfo() = default;
 
-    void add(VariablePrintInfo* v);
+    void add(std::string name, VariablePrintInfo v);
     void clear();
-    VariablePrintInfo* operator[](uint i) const;
+    VariablePrintInfo& operator[](uint i);
     size_t size() const;
-    bool isEmpty() const;
```

Comment:
We don't need **isEmpty()**, since the output variables are built at compile time and their number is fixed